src/test/resources/application-test.conf

```diff
@@ -9,8 +9,8 @@ party-process {
   confirm-token-timeout = "3 seconds"
   signature-validation-enabled = true
   destination-mails= "mail@mail.com"
-  eu_list_of_trusted_lists_url = "eu_list_of_trusted_lists_url"
-  eu_official_journal_url   = "eu_official_journal_url"
+  eu_list_of_trusted_lists_url = "https://ec.europa.eu/tools/lotl/eu-lotl.xml"
+  eu_official_journal_url = "https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=uriserv:OJ.C_.2019.276.01.0001.01.ENG"
```

Comment:
are these endpoints actually used during tests?

Reply:
These are useful for the SignatureSpec test, so explicitly putting them here simplifies the test usage.